### PR TITLE
simplify js error

### PR DESCRIPTION
**Pull request by benrudolph**

### CODE REVIEW

**Comment by benrudolph** [comment]
enchantress: @millerdev

File: dimagi/utils/logging.py [hunk 1]
```diff
@@ -40,17 +40,13 @@ def notify_exception(request, message=None, details=None):
     )
 
 
-def notify_js_exception(request, message=None, filename=None, stack=None, line=None, count=None):
+def notify_js_exception(request, message=None, details=None):
```
Comment (dannyroberts) [inline dimagi/utils/logging.py]: did you check for all usages of `notify_js_exception` other than your own?

Comment (benrudolph) [inline dimagi/utils/logging.py]: i wrote the function for this purpose haha it's only used in /jserror

Comment (benrudolph) [inline dimagi/utils/logging.py]: though i'm going to update the agent thing

**Pull request merged**